projects/examples/src/app/components/action-menu/search/action-menu-search-example.component.ts

```diff
@@ -95,6 +95,11 @@ export class ActionMenuSearchExampleComponent<R extends Record, T extends Handle
                 },
             ],
         },
+        {
```

Comment:
There should be some text explanation as to what is being exemplified here. We have to make sure we're not trying to show too many features in one example